Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1780,8 +1782,65 @@ def create_clickable_url(url):
     return '[{}]({})'.format(url, url)
 
 
+class JsonTransformer:
+    def __init__(self, flatten=False, keys_lst=None, is_nested=False, func=None):
+        """
+        Constructor for JsonTransformer
+
+        :type flatten: ``bool``
+        :param flatten:  Should we flatten the json using `flattenCell` (the default behavior)
+
+        :type keys_lst: ``List[str]``
+        :param keys_lst: a list of relevant keys list from the json
+
+        :type is_nested: ``bool``
+        :param is_nested: Whether to search in nested keys or not
+
+        :type func: ``Callable``
+        :param func: A function to parse the json
+        """
+        if keys_lst is None:
+            keys_lst = []
+        self.keys_set = set(keys_lst)
+        self.is_nested = is_nested
+        self.func = func
+        self.flatten = flatten
+
+    def json_to_str(self, json_input, is_pretty=True):
+        if self.func:
+            return self.func(json_input)
+        if isinstance(json_input, STRING_TYPES):
+            return json_input
+        if isinstance(json_input, list):
+            return flattenCell(json_input, is_pretty)
+        if self.flatten:
+            return '\n'.join(
+                ['{key}: {val}'.format(key=k, val=flattenCell(v, is_pretty)) for k, v in json_input.items()])
+        str_lst = []
```

Comment:
Separating main logic from the initial edge-case checks (code readability).
```suggestion

        str_lst = []
```